spec.emu

```diff
@@ -55,9 +35,20 @@ contributors: Jordan Harband
               1. Append code unit U+005C (REVERSE SOLIDUS) to _escapedList_.
               1. Append code unit U+0078 (LATIN SMALL LETTER X) to _escapedList_.
               1. Append code unit U+0033 (DIGIT THREE) to _escapedList_.
+              1. Append _c_ to _escapedList_.
```

Comment:
You can eliminate this whole branch if you add the digits to `_punctuators_`.

Reply:
This branch is also guarded on `If _escapedList_ is empty`, so no, you can't.